src/azure-cli/azure/cli/command_modules/acs/_roleassignments.py

```diff
@@ -0,0 +1,168 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import time
+import uuid
+
+from azure.cli.core.azclierror import AzCLIError
+from azure.cli.core.profiles import ResourceType, get_sdk
+from azure.core.exceptions import HttpResponseError
+from azure.graphrbac.models import GetObjectsParameters
+from knack.log import get_logger
+from msrestazure.azure_exceptions import CloudError
+
+from ._client_factory import (get_auth_management_client,
+                              get_graph_rbac_management_client)
+
+logger = get_logger(__name__)
+
+
+def _get_object_stubs(graph_client, assignees):
```

Comment:
For reference, [this](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/custom.py#L1552) is the corresponding implementation in `custom.py`.